src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -617,7 +632,7 @@ protected DynamoModel(IStartConfiguration config)
                     }
                     catch (Exception ex)
                     {
-                        Logger.Log(ex.Message);                       
```

Comment:
seems there a lot of line endings changes that make tracking the real changes more difficult. Any idea whats going on there?